curriculum/challenges/english/01-responsive-web-design/css-variables-skyline/part-018.md

```diff
@@ -7,152 +7,59 @@ dashedName: part-18
 
 # --description--
 
-Your first building looks pretty good now. Let's make some more! Nest three new `div` elements in the `background-buildings` container and give them the classes of `bb2`, `bb3`, and `bb4` in that order. These will be three more buildings for the background.
+Your first building looks pretty good now. Let's make some more! Nest three new `div` elements in the `background-buildings` container and give them the classes of `.bb2`, `.bb3`, and `.bb4` in that order. These will be three more buildings for the background.
```

Comment:
Should classes be with the dots here? They seem to be referring to the class names rather than the selectors.

Reply:
Quite right. I will remove those.